mypy/messages.py

```diff
@@ -528,6 +528,7 @@ def incompatible_argument(self,
                 arg_type_str = '*' + arg_type_str
             elif arg_kind == ARG_STAR2:
                 arg_type_str = '**' + arg_type_str
+                notes.append("Use TypedDict or Dict[str, Any] for unpacked **kwargs")
```

Comment:
Hey! You will also have to modify the tests affected by this change :) Cheers!

Reply:
yea, its a work in progress